src/proto/Ethereum.proto

```diff
@@ -3,6 +3,32 @@ syntax = "proto3";
 package TW.Ethereum.Proto;
 option java_package = "wallet.core.jni.proto";
 
+message TransferContract {
```

Comment:
I prefer to nest them under one message, see `Comos.proto`, this will reduce exposing too many proto types to swift/java

Reply:
OK, done